packages/fxa-auth-server/fxa-oauth-server/lib/validators.js

```diff
@@ -68,3 +67,8 @@ exports.jwe = Joi.string()
   .regex(
     /^[A-Za-z0-9-_]+\.[A-Za-z0-9-_]*\.[A-Za-z0-9-_]+\.[A-Za-z0-9-_]+\.[A-Za-z0-9-_]+$/
   );
+
+exports.accessToken = Joi.alternatives().try([
+  exports.token,
+  exports.assertion,
```

Comment:
nit: this will allow BrowserID assertions to validate as valid access tokens. Probably harmless, because they will fail JWT verification, but it may be worth adding an explicit test to that effect.

Reply:
Done